i18n/MonotoneMuseum/ko_kr.yml

```
@@ -25,65 +25,538 @@ ko_kr:
+        - |-
+          【마법폭주】
+          마법의 힘이 폭주해 더는 누구도 감당할 수 없다. 세계는 끝난다. 사라진다. 이 씬 동안 「구분 : 술」의 특기를 사용한 판정의 펌블치가 +6이 된다.
+        - |-
+          【자기사】
+          거대한 올풀림이 발생해 치명적인 대미지를 입힌다. 가장 박탈치가 높은 PC 한 명이 전투불능이 된다. 다수의 PC가 해당하는 경우 GM이 랜덤으로 결정한다.
+        - |-
+          【허무의 응보】
+          실잣는이에 의해 계속해서 일그러져왔던 인과가 역습한다. 이 씬에서 《허구출현》을 사용한 경우 대가에 더해 10D6의 실대미지를 입는다. 이 대미지는 경감할 수 없으며 대신 받을 수도 없다.
+        - |-
+          【허구의 끝】
+          신이 버린 것인가 신을 버린 것인가. 이탈의 끝에 세계가 끝장난다. 당신은 이 씬에서 이탈능력을 사용한 경우 상승한 박탈치의 10배만큼 실 대미지를 입는다. 이 대미지는 경감할 수 없으며 대신 받을 수도 없다.
+        - |-
+          【인격교대】
+          혼, 엄밀히 말하면 혼의 배후에 있는 무언가가 뒤바뀐다. 왼쪽의 플레이어에게 캐릭터 시트를 넘기고 이 신 동안 그 플레이어가 당신의 PC를 연기한다. GM이 인정한다면 GM도 NPC를 바꾼다.
+        - |-
+          【인과의 실】
+          비웃는 소리가 들린다. 「이탈의 결과는 공유하자고？」 PC 전원의 박탈치의 평균을 내 (소수점 올림) 전원의 박탈치를 그 계산결과로 바꿔쓴다.
+        - |-
+          【검에 죽다】
+          사람을 벌한 응보를, 검에는 검의 대가를. 이 씬에서 공격을 하고 펌블했을 경우 5D6의 실데미지를 입는다.
+        - |-
+          【비밀폭로】
+          등장해있는 모든 실잣는이, 이형, 가람은 숨기고싶은 비밀(그 캐릭터의 주관으로 OK) 하나를 밝힌다. 그러지 못할 경우 10D6의 실대미지를 받는다. 이 대미지는 경감할 수도 대신 받을 수도 없다.
+        - |-
+          【황폐의 왕】
+          깨달았을 때는 모든 캐릭터가 허무의 황야의 황폐의 왕의 몸 위에 있다. 모든 캐릭터는 그대로 허무에 삼켜져 5D6의 실데미지를 입는다. 이 데미지는 경감할 수도 대신 받을 수도 없다. 황폐의 왕이 당신들에게 어떤 영향을 미칠지는 GM이 정한다.
+      OT2:
+        name: 징조표ver2.0
+        items:
+        - |-
+          【신념 상실】
+          당신의 정신이 당신의 과거를 부정한다. 당신은 [출신]과 그에 해당하는 기억을 그 씬 동안 상실한다. 특성으로 인한 효과는 잃지 않지만 그것을 인식하는 것은 불가능하다.
+        - |-
+          【졸도】
+          더 이상 당신은 이탈의 부담을 버티는 것이 불가능하다. 그것이 세계에 거스른 대가이다. 당신은 [전투불능]이 된다. 이미 [전투불능]일 경우 [사망]한다.
+        - |-
+          【육체 붕괴】
+          유리가 깨지는 소리가 나며, 당신도 깨져간다. 당신은 2D6+캐릭터레벨 점의 【HP】를 잃는다. 
+        - |-
+          【방심】
+          당신의 마음에 커다란 충격이 일었다. 마음은 가람을 향해 일그러지고, 눈동자는 색 없는 악몽을 본다. 당신은 배드스테이터스 [방심]을 받는다.
+        - |-
+          【중압】
+          어디에선가 당신을 비웃는 목소리가 들려온다. 「이리하여 이정표를 거스른 어리석은 자는 쇠사슬에 얽매여 모진 고문을 받는 것이었습니다」. 당신은 배드스테이터스 [중압]을 받는다.
+        - |-
+          【현재 상실】
+          현재 갖고있는 파트너를 하나 상실한다. 당신은 무언가 잃었다는 것은 눈치채나 그 존재에 대한 것은 기억해낼 수 없다. 이 효과는 그 공연 동안 지속된다. 물론, 새롭게 인간관계를 구축하는 것은 가능하다.
+        - |-
+          【마비】
+          이형화한 육체의 변이에, 영혼이 버티지 못한다. 몸이 저리고 떨림이 멎지 않는다. 당신은 배드스테이터스 [마비]를 받는다.
+        - |-
+          【중독】
+          의식 깊은 곳에서 이형화가 진행되어 당신의 육체를 좀먹기 시작한다. 당신은 배드스테이터스 [사독]5를 받는다.
+        - |-
+          【색채 상실】
+          당신의 몸에서 색이 사라져 탁한 백색에 가까워진다. 옷이나 화장으로 숨기는 것은 가능하나 들키면 이단심문은 피할 수 없을테지. 당신의 박탈치를 +1d6한다 (이 효과로는 징조표가 굴러가지 않는다)
+        - |-
+          【이유 상실】
+          당신의 정신이 당신의 현재를 부정한다. 당신은 [경험]과 그것에 관한 기억을 그 씬 동안 잊는다. 특징에 의한 효과는 사라지지 않지만 그것을 인식하는 것은 불가능하다.
+        - |-
+          【존재 상실】
+          문득 연기처럼 당신의 모습이 사라진다. 당신의 존재는 잠깐동안 이 세계에서 소실된다. 당신의 박탈치를 +1D6한다 (이 효과로는 징조표를 굴리지 않는다)
+        - |-
+          【이탈의 대가】
+          이탈능력을 사용한 결과로 징조표를 굴리게 되었을 경우에만 적용한다 (그렇지 않을 경우 다시 굴린다). 이 표를 굴리는 원인이 된 이탈능력을 이 라운드 (전투중이 아니라면 씬 동안) 동안 더 이상 사용할 수 없다. 
```

The trailing space should be removed.
```suggestion
          이탈능력을 사용한 결과로 징조표를 굴리게 되었을 경우에만 적용한다 (그렇지 않을 경우 다시 굴린다). 이 표를 굴리는 원인이 된 이탈능력을 이 라운드 (전투중이 아니라면 씬 동안) 동안 더 이상 사용할 수 없다.
```